samples/javascript_nodejs/13.core-bot/index.js

```diff
@@ -36,7 +36,9 @@ const BOOKING_DIALOG = 'bookingDialog';
 
 const credentialsFactory = new ConfigurationServiceClientCredentialFactory({
     MicrosoftAppId: process.env.MicrosoftAppId,
-    MicrosoftAppPassword: process.env.MicrosoftAppPassword
+    MicrosoftAppPassword: process.env.MicrosoftAppPassword,
+    MicrosoftAppType: process.env.MicrosoftAppType,
```

Comment:
Not too drastic, but I think the AppType could be first as on the .env file. It's ok if you decide to not change that.

Reply:
I followed the order of the parameters in the class constructor. I think we should keep it like that.